### PR TITLE
Add FT240X EEPROM. Remove FtdiMpsse impl for FT232R because it is unsupported

### PR DESCRIPTION
**Pull request by Farmadupe**
* Add FT240X EEPROM impls
* Factorize out mpsse functions from impl_boilerplate_for! macro, as not all devices support mpsse (FT240X, FT232R)
* Remove impl of FtdiMpsse for Ft232R, as the docs say it is not supported (breaking change)
* Inhibit tests for set_vid_pid functions on windows, as those functions are not available.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -98,14 +98,16 @@ mod errors;
 pub use errors::{DeviceTypeError, EepromStringsError, EepromValueError, FtStatus, TimeoutError};
 
 mod mpsse;
+use libftd2xx_ffi::FT_EEPROM_X_SERIES;
```
Comment (newAM) [inline src/lib.rs:101]: There's an import for all `libftd2xx_ffi` types below this, lump it in with those.

File: src/types.rs [hunk 2]
```diff
@@ -3,14 +3,14 @@
...
 use libftd2xx_ffi::{
-    FT_X_SERIES_CBUS_BCD_CHARGER, FT_X_SERIES_CBUS_BCD_CHARGER_N, FT_X_SERIES_CBUS_BITBANG_RD,
-    FT_X_SERIES_CBUS_BITBANG_WR, FT_X_SERIES_CBUS_CLK12, FT_X_SERIES_CBUS_CLK24,
-    FT_X_SERIES_CBUS_CLK6, FT_X_SERIES_CBUS_DRIVE_0, FT_X_SERIES_CBUS_DRIVE_1,
-    FT_X_SERIES_CBUS_I2C_RXF, FT_X_SERIES_CBUS_I2C_TXE, FT_X_SERIES_CBUS_IOMODE,
-    FT_X_SERIES_CBUS_KEEP_AWAKE, FT_X_SERIES_CBUS_PWREN, FT_X_SERIES_CBUS_RXLED,
-    FT_X_SERIES_CBUS_SLEEP, FT_X_SERIES_CBUS_TIMESTAMP, FT_X_SERIES_CBUS_TRISTATE,
-    FT_X_SERIES_CBUS_TXDEN, FT_X_SERIES_CBUS_TXLED, FT_X_SERIES_CBUS_TXRXLED,
-    FT_X_SERIES_CBUS_VBUS_SENSE,
+    FT_EEPROM_X_SERIES, FT_X_SERIES_CBUS_BCD_CHARGER, FT_X_SERIES_CBUS_BCD_CHARGER_N,
```
Comment (newAM) [inline src/types.rs:6]: `FT_EEPROM_X_SERIES` should be with the other `FT_EEPROM_`'s

**Review by newAM — CHANGES_REQUESTED** [review]
Thanks!  Just a couple minor changes.

**Comment by Farmadupe** [comment]
Comments addressed

**Review by newAM — APPROVED** [review]
Thanks!  Do you want a release with these changes?

**Comment by Farmadupe** [comment]
A new release would be appreciated!

**Comment by newAM** [comment]
Done in 0.33.0!

**Pull request merged**